### ISSUE TITLE
Not all AddOns are loaded

### ISSUE THREAD

**Issue by Riscon** [casperstorm/ajour#266]
**Describe the bug**
When you open Ajour or press the button "Refresh" it won't always load / display all of your addons inside your wow folder.

**To Reproduce**
1. Open Ajour and check your number of addons loaded on top of the programm. Ex.: 31 retail addons loaded
2. Click on "Refresh" and check again. Ex. 35 retail addons loaded.

**Expected behavior**
I expect that Ajour always scans for all my addons inside my wow folder.

**Screenshots**
To show better show the bug, I've recorded a video.
[https://streamable.com/7jng38](https://streamable.com/7jng38)

**Software involved**
Please complete the following information:
 - OS: Windows 10
 - Ajour version: 0.4.1
 - Flavor (retail) and version: 9.0.1

**Log Output**

> 22:48:33.800 [ajour][INFO] Ajour 0.4.1 has started.
22:48:33.801 [ajour::gui][DEBUG] antialiasing: true
22:48:33.815 [ajour_core::config][DEBUG] loading config
22:48:33.815 [ajour_core::utility][DEBUG] checking for application update
22:48:33.815 [ajour_core::theme][DEBUG] loading user themes
22:48:33.817 [ajour_core::fs::theme][DEBUG] loaded 0 user themes
22:48:34.318 [ajour::gui::update][DEBUG] Message::Parse
22:48:34.318 [ajour::gui::update][DEBUG] config loaded:
Config {
    wow: Wow {
        directory: Some(
            "D:\\Games\\World of Warcraft",
        ),
        flavor: Retail,
    },
    addons: Addons {
        ignored: {
            Retail: [],
        },
        release_channels: {
            Retail: {},
        },
    },
    theme: None,
    column_config: V1 {
        local_version_width: 150,
        remote_version_width: 150,
        status_width: 85,
    },
    window_size: Some(
        (
            1920,
            1027,
        ),
    ),
    scale: None,
    backup_directory: None,
}
22:48:34.318 [ajour::gui::update][DEBUG] preparing to parse addons in "D:\\Games\\World of Warcraft\\_retail_\\Interface/AddOns"
22:48:34.319 [ajour::gui::update][DEBUG] preparing to parse addons in "D:\\Games\\World of Warcraft\\_classic_\\Interface/AddOns"
22:48:34.319 [ajour::gui::update][DEBUG] Message::ThemesLoaded(0 themes)
22:48:34.319 [ajour::gui::update][DEBUG] Message::NeedsUpdate(None)
22:48:34.319 [ajour_core::parse][DEBUG] retail - parsing addons folder
22:48:34.319 [ajour_core::parse][DEBUG] classic - parsing addons folder
22:48:34.319 [ajour_core::parse][DEBUG] retail - 102 folders in AddOns directory to parse
22:48:34.340 [ajour::gui::update][ERROR] Message::ParsedAddons(classic) - Addon directory not found: "D:\\Games\\World of Warcraft\\_classic_\\Interface/AddOns"
22:48:34.365 [ajour_core::parse][DEBUG] retail - 102 fingerprints: 102 cached, 0 calculated, 0 added, 0 removed
22:48:34.408 [ajour_core::parse][DEBUG] retail - 101 successfully parsed from '.toc'
22:48:34.408 [ajour_core::parse][DEBUG] retail - 2 addons with tukui id
22:48:35.196 [ajour_core::parse][DEBUG] retail - 2 addons applied with tukui id package metadata
22:48:35.196 [ajour_core::parse][DEBUG] retail - 99 unique fingerprints to check against curse api
22:48:35.339 [ajour::gui::update][DEBUG] Message::CatalogDownloaded(8635 addons in catalog)
22:48:35.520 [ajour_core::parse][TRACE] retail - partial fingerprint found:
	Curse ID: 3073785
	File Name: FarmHud-8.4.9-release-nolib
	Modules (Name - Local Fingerprint - Remote Fingerprint)
	FarmHud_QuestArrow - 444818479 - 444818479
	FarmHud - 1734683697 - 4117404573
22:48:35.520 [ajour_core::parse][DEBUG] retail - 31 exact fingerprint matches against curse api
22:48:35.521 [ajour_core::parse][DEBUG] retail - 31 addons applied with fingerprint metadata
22:48:35.521 [ajour_core::parse][DEBUG] retail - 31 addons with curse id
22:48:35.923 [ajour_core::parse][DEBUG] retail - 31 addons applied with curse id package metadata
22:48:35.923 [ajour_core::parse][DEBUG] retail - 33 addons successfully parsed
22:48:35.925 [ajour::gui::update][DEBUG] Message::ParsedAddons(retail, 33 addons)
22:50:00.817 [ajour::gui::update][DEBUG] Interaction::Refresh
22:50:00.817 [ajour_core::config][DEBUG] loading config
22:50:00.819 [ajour::gui::update][DEBUG] Message::Parse
22:50:00.819 [ajour::gui::update][DEBUG] config loaded:
Config {
    wow: Wow {
        directory: Some(
            "D:\\Games\\World of Warcraft",
        ),
        flavor: Retail,
    },
    addons: Addons {
        ignored: {
            Retail: [],
        },
        release_channels: {
            Retail: {},
        },
    },
    theme: None,
    column_config: V1 {
        local_version_width: 150,
        remote_version_width: 150,
        status_width: 85,
    },
    window_size: Some(
        (
            645,
            544,
        ),
    ),
    scale: None,
    backup_directory: None,
}
22:50:00.819 [ajour::gui::update][DEBUG] preparing to parse addons in "D:\\Games\\World of Warcraft\\_retail_\\Interface/AddOns"
22:50:00.819 [ajour::gui::update][DEBUG] preparing to parse addons in "D:\\Games\\World of Warcraft\\_classic_\\Interface/AddOns"
22:50:00.819 [ajour_core::parse][DEBUG] retail - parsing addons folder
22:50:00.819 [ajour_core::parse][DEBUG] classic - parsing addons folder
22:50:00.820 [ajour_core::parse][DEBUG] retail - 102 folders in AddOns directory to parse
22:50:00.820 [ajour::gui::update][ERROR] Message::ParsedAddons(classic) - Addon directory not found: "D:\\Games\\World of Warcraft\\_classic_\\Interface/AddOns"
22:50:00.823 [ajour_core::parse][DEBUG] retail - 102 fingerprints: 102 cached, 0 calculated, 0 added, 0 removed
22:50:00.857 [ajour_core::parse][DEBUG] retail - 101 successfully parsed from '.toc'
22:50:00.857 [ajour_core::parse][DEBUG] retail - 2 addons with tukui id
22:50:01.609 [ajour_core::parse][DEBUG] retail - 2 addons applied with tukui id package metadata
22:50:01.610 [ajour_core::parse][DEBUG] retail - 99 unique fingerprints to check against curse api
22:50:01.983 [ajour_core::parse][DEBUG] retail - 34 exact fingerprint matches against curse api
22:50:01.983 [ajour_core::parse][DEBUG] retail - 34 addons applied with fingerprint metadata
22:50:01.983 [ajour_core::parse][DEBUG] retail - 34 addons with curse id
22:50:02.377 [ajour_core::parse][DEBUG] retail - 34 addons applied with curse id package metadata
22:50:02.377 [ajour_core::parse][DEBUG] retail - 36 addons successfully parsed
22:50:02.379 [ajour::gui::update][DEBUG] Message::ParsedAddons(retail, 36 addons)
22:50:20.891 [ajour::gui::update][DEBUG] Interaction::Refresh
22:50:20.892 [ajour_core::config][DEBUG] loading config
22:50:20.893 [ajour::gui::update][DEBUG] Message::Parse
22:50:20.893 [ajour::gui::update][DEBUG] config loaded:
Config {
    wow: Wow {
        directory: Some(
            "D:\\Games\\World of Warcraft",
        ),
        flavor: Retail,
    },
    addons: Addons {
        ignored: {
            Retail: [],
        },
        release_channels: {
            Retail: {},
        },
    },
    theme: None,
    column_config: V1 {
        local_version_width: 150,
        remote_version_width: 150,
        status_width: 85,
    },
    window_size: Some(
        (
            645,
            544,
        ),
    ),
    scale: None,
    backup_directory: None,
}
22:50:20.894 [ajour::gui::update][DEBUG] preparing to parse addons in "D:\\Games\\World of Warcraft\\_retail_\\Interface/AddOns"
22:50:20.894 [ajour::gui::update][DEBUG] preparing to parse addons in "D:\\Games\\World of Warcraft\\_classic_\\Interface/AddOns"
22:50:20.894 [ajour_core::parse][DEBUG] retail - parsing addons folder
22:50:20.894 [ajour_core::parse][DEBUG] classic - parsing addons folder
22:50:20.894 [ajour_core::parse][DEBUG] retail - 102 folders in AddOns directory to parse
22:50:20.895 [ajour::gui::update][ERROR] Message::ParsedAddons(classic) - Addon directory not found: "D:\\Games\\World of Warcraft\\_classic_\\Interface/AddOns"
22:50:20.898 [ajour_core::parse][DEBUG] retail - 102 fingerprints: 102 cached, 0 calculated, 0 added, 0 removed
22:50:20.929 [ajour_core::parse][DEBUG] retail - 101 successfully parsed from '.toc'
22:50:20.929 [ajour_core::parse][DEBUG] retail - 2 addons with tukui id
22:50:21.692 [ajour_core::parse][DEBUG] retail - 2 addons applied with tukui id package metadata
22:50:21.692 [ajour_core::parse][DEBUG] retail - 99 unique fingerprints to check against curse api
22:50:22.028 [ajour_core::parse][TRACE] retail - partial fingerprint found:
	Curse ID: 3073785
	File Name: FarmHud-8.4.9-release-nolib
	Modules (Name - Local Fingerprint - Remote Fingerprint)
	FarmHud_QuestArrow - 444818479 - 444818479
	FarmHud - 1734683697 - 4117404573
22:50:22.028 [ajour_core::parse][DEBUG] retail - 30 exact fingerprint matches against curse api
22:50:22.028 [ajour_core::parse][DEBUG] retail - 30 addons applied with fingerprint metadata
22:50:22.028 [ajour_core::parse][DEBUG] retail - 30 addons with curse id
22:50:22.642 [ajour_core::parse][DEBUG] retail - 30 addons applied with curse id package metadata
22:50:22.642 [ajour_core::parse][DEBUG] retail - 32 addons successfully parsed
22:50:22.644 [ajour::gui::update][DEBUG] Message::ParsedAddons(retail, 32 addons)
22:50:24.584 [ajour::gui::update][DEBUG] Interaction::Refresh
22:50:24.584 [ajour_core::config][DEBUG] loading config
22:50:24.586 [ajour::gui::update][DEBUG] Message::Parse
22:50:24.586 [ajour::gui::update][DEBUG] config loaded:
Config {
    wow: Wow {
        directory: Some(
            "D:\\Games\\World of Warcraft",
        ),
        flavor: Retail,
    },
    addons: Addons {
        ignored: {
            Retail: [],
        },
        release_channels: {
            Retail: {},
        },
    },
    theme: None,
    column_config: V1 {
        local_version_width: 150,
        remote_version_width: 150,
        status_width: 85,
    },
    window_size: Some(
        (
            645,
            544,
        ),
    ),
    scale: None,
    backup_directory: None,
}
22:50:24.586 [ajour::gui::update][DEBUG] preparing to parse addons in "D:\\Games\\World of Warcraft\\_retail_\\Interface/AddOns"
22:50:24.586 [ajour::gui::update][DEBUG] preparing to parse addons in "D:\\Games\\World of Warcraft\\_classic_\\Interface/AddOns"
22:50:24.586 [ajour_core::parse][DEBUG] retail - parsing addons folder
22:50:24.586 [ajour_core::parse][DEBUG] classic - parsing addons folder
22:50:24.587 [ajour_core::parse][DEBUG] retail - 102 folders in AddOns directory to parse
22:50:24.587 [ajour::gui::update][ERROR] Message::ParsedAddons(classic) - Addon directory not found: "D:\\Games\\World of Warcraft\\_classic_\\Interface/AddOns"
22:50:24.590 [ajour_core::parse][DEBUG] retail - 102 fingerprints: 102 cached, 0 calculated, 0 added, 0 removed
22:50:24.624 [ajour_core::parse][DEBUG] retail - 101 successfully parsed from '.toc'
22:50:24.625 [ajour_core::parse][DEBUG] retail - 2 addons with tukui id
22:50:25.386 [ajour_core::parse][DEBUG] retail - 2 addons applied with tukui id package metadata
22:50:25.387 [ajour_core::parse][DEBUG] retail - 99 unique fingerprints to check against curse api
22:50:25.785 [ajour_core::parse][TRACE] retail - partial fingerprint found:
	Curse ID: 2888005
	File Name: AtlasLoot-v8.12.00-nolib
	Modules (Name - Local Fingerprint - Remote Fingerprint)
	AtlasLoot_BattleforAzeroth - 1923473088 - 1923473088
	AtlasLoot_BurningCrusade - 2475028490 - 2475028490
	AtlasLoot_Cataclysm - 1273879134 - 1273879134
	AtlasLoot_Classic - 1259917105 - 1259917105
	AtlasLoot_Collections - 3427175467 - 3427175467
	AtlasLoot_Crafting - 1634075948 - 1634075948
	AtlasLoot_Factions - 3089872706 - 3089872706
	AtlasLoot_Legion - 2435882017 - 2435882017
	AtlasLoot_MistsofPandaria - 1065327854 - 1065327854
	AtlasLoot_Options - 1387568472 - 1387568472
	AtlasLoot_PvP - 1400949647 - 1400949647
	AtlasLoot_WarlordsofDraenor - 3169191016 - 3169191016
	AtlasLoot_WorldEvents - 3907889742 - 3907889742
	AtlasLoot_WrathoftheLichKing - 683334673 - 683334673
	AtlasLoot - 1541435651 - 3486052583
22:50:25.785 [ajour_core::parse][DEBUG] retail - 32 exact fingerprint matches against curse api
22:50:25.785 [ajour_core::parse][DEBUG] retail - 32 addons applied with fingerprint metadata
22:50:25.786 [ajour_core::parse][DEBUG] retail - 32 addons with curse id
22:50:26.367 [ajour_core::parse][DEBUG] retail - 32 addons applied with curse id package metadata
22:50:26.367 [ajour_core::parse][DEBUG] retail - 34 addons successfully parsed
22:50:26.368 [ajour::gui::update][DEBUG] Message::ParsedAddons(retail, 34 addons)
22:50:30.446 [ajour::gui::update][DEBUG] Interaction::Refresh
22:50:30.446 [ajour_core::config][DEBUG] loading config
22:50:30.449 [ajour::gui::update][DEBUG] Message::Parse
22:50:30.449 [ajour::gui::update][DEBUG] config loaded:
Config {
    wow: Wow {
        directory: Some(
            "D:\\Games\\World of Warcraft",
        ),
        flavor: Retail,
    },
    addons: Addons {
        ignored: {
            Retail: [],
        },
        release_channels: {
            Retail: {},
        },
    },
    theme: None,
    column_config: V1 {
        local_version_width: 150,
        remote_version_width: 150,
        status_width: 85,
    },
    window_size: Some(
        (
            645,
            544,
        ),
    ),
    scale: None,
    backup_directory: None,
}
22:50:30.449 [ajour::gui::update][DEBUG] preparing to parse addons in "D:\\Games\\World of Warcraft\\_retail_\\Interface/AddOns"
22:50:30.449 [ajour::gui::update][DEBUG] preparing to parse addons in "D:\\Games\\World of Warcraft\\_classic_\\Interface/AddOns"
22:50:30.449 [ajour_core::parse][DEBUG] retail - parsing addons folder
22:50:30.449 [ajour_core::parse][DEBUG] classic - parsing addons folder
22:50:30.450 [ajour_core::parse][DEBUG] retail - 102 folders in AddOns directory to parse
22:50:30.453 [ajour::gui::update][ERROR] Message::ParsedAddons(classic) - Addon directory not found: "D:\\Games\\World of Warcraft\\_classic_\\Interface/AddOns"
22:50:30.453 [ajour_core::parse][DEBUG] retail - 102 fingerprints: 102 cached, 0 calculated, 0 added, 0 removed
22:50:30.483 [ajour_core::parse][DEBUG] retail - 101 successfully parsed from '.toc'
22:50:30.483 [ajour_core::parse][DEBUG] retail - 2 addons with tukui id
22:50:31.255 [ajour_core::parse][DEBUG] retail - 2 addons applied with tukui id package metadata
22:50:31.255 [ajour_core::parse][DEBUG] retail - 99 unique fingerprints to check against curse api
22:50:31.552 [ajour_core::parse][TRACE] retail - partial fingerprint found:
	Curse ID: 2888005
	File Name: AtlasLoot-v8.12.00-nolib
	Modules (Name - Local Fingerprint - Remote Fingerprint)
	AtlasLoot_BattleforAzeroth - 1923473088 - 1923473088
	AtlasLoot_BurningCrusade - 2475028490 - 2475028490
	AtlasLoot_Cataclysm - 1273879134 - 1273879134
	AtlasLoot_Classic - 1259917105 - 1259917105
	AtlasLoot_Collections - 3427175467 - 3427175467
	AtlasLoot_Crafting - 1634075948 - 1634075948
	AtlasLoot_Factions - 3089872706 - 3089872706
	AtlasLoot_Legion - 2435882017 - 2435882017
	AtlasLoot_MistsofPandaria - 1065327854 - 1065327854
	AtlasLoot_Options - 1387568472 - 1387568472
	AtlasLoot_PvP - 1400949647 - 1400949647
	AtlasLoot_WarlordsofDraenor - 3169191016 - 3169191016
	AtlasLoot_WorldEvents - 3907889742 - 3907889742
	AtlasLoot_WrathoftheLichKing - 683334673 - 683334673
	AtlasLoot - 1541435651 - 3486052583
22:50:31.552 [ajour_core::parse][DEBUG] retail - 29 exact fingerprint matches against curse api
22:50:31.552 [ajour_core::parse][DEBUG] retail - 29 addons applied with fingerprint metadata
22:50:31.552 [ajour_core::parse][DEBUG] retail - 29 addons with curse id
22:50:31.968 [ajour_core::parse][DEBUG] retail - 29 addons applied with curse id package metadata
22:50:31.969 [ajour_core::parse][DEBUG] retail - 31 addons successfully parsed
22:50:31.970 [ajour::gui::update][DEBUG] Message::ParsedAddons(retail, 31 addons)
22:50:33.428 [ajour::gui::update][DEBUG] Interaction::Refresh
22:50:33.428 [ajour_core::config][DEBUG] loading config
22:50:33.430 [ajour::gui::update][DEBUG] Message::Parse
22:50:33.430 [ajour::gui::update][DEBUG] config loaded:
Config {
    wow: Wow {
        directory: Some(
            "D:\\Games\\World of Warcraft",
        ),
        flavor: Retail,
    },
    addons: Addons {
        ignored: {
            Retail: [],
        },
        release_channels: {
            Retail: {},
        },
    },
    theme: None,
    column_config: V1 {
        local_version_width: 150,
        remote_version_width: 150,
        status_width: 85,
    },
    window_size: Some(
        (
            645,
            544,
        ),
    ),
    scale: None,
    backup_directory: None,
}
22:50:33.430 [ajour::gui::update][DEBUG] preparing to parse addons in "D:\\Games\\World of Warcraft\\_retail_\\Interface/AddOns"
22:50:33.430 [ajour::gui::update][DEBUG] preparing to parse addons in "D:\\Games\\World of Warcraft\\_classic_\\Interface/AddOns"
22:50:33.431 [ajour_core::parse][DEBUG] retail - parsing addons folder
22:50:33.431 [ajour_core::parse][DEBUG] classic - parsing addons folder
22:50:33.431 [ajour_core::parse][DEBUG] retail - 102 folders in AddOns directory to parse
22:50:33.432 [ajour::gui::update][ERROR] Message::ParsedAddons(classic) - Addon directory not found: "D:\\Games\\World of Warcraft\\_classic_\\Interface/AddOns"
22:50:33.435 [ajour_core::parse][DEBUG] retail - 102 fingerprints: 102 cached, 0 calculated, 0 added, 0 removed
22:50:33.469 [ajour_core::parse][DEBUG] retail - 101 successfully parsed from '.toc'
22:50:33.469 [ajour_core::parse][DEBUG] retail - 2 addons with tukui id
22:50:34.219 [ajour_core::parse][DEBUG] retail - 2 addons applied with tukui id package metadata
22:50:34.220 [ajour_core::parse][DEBUG] retail - 99 unique fingerprints to check against curse api
22:50:34.456 [ajour_core::parse][TRACE] retail - partial fingerprint found:
	Curse ID: 2888005
	File Name: AtlasLoot-v8.12.00-nolib
	Modules (Name - Local Fingerprint - Remote Fingerprint)
	AtlasLoot_BattleforAzeroth - 1923473088 - 1923473088
	AtlasLoot_BurningCrusade - 2475028490 - 2475028490
	AtlasLoot_Cataclysm - 1273879134 - 1273879134
	AtlasLoot_Classic - 1259917105 - 1259917105
	AtlasLoot_Collections - 3427175467 - 3427175467
	AtlasLoot_Crafting - 1634075948 - 1634075948
	AtlasLoot_Factions - 3089872706 - 3089872706
	AtlasLoot_Legion - 2435882017 - 2435882017
	AtlasLoot_MistsofPandaria - 1065327854 - 1065327854
	AtlasLoot_Options - 1387568472 - 1387568472
	AtlasLoot_PvP - 1400949647 - 1400949647
	AtlasLoot_WarlordsofDraenor - 3169191016 - 3169191016
	AtlasLoot_WorldEvents - 3907889742 - 3907889742
	AtlasLoot_WrathoftheLichKing - 683334673 - 683334673
	AtlasLoot - 1541435651 - 3486052583
22:50:34.456 [ajour_core::parse][DEBUG] retail - 33 exact fingerprint matches against curse api
22:50:34.457 [ajour_core::parse][DEBUG] retail - 33 addons applied with fingerprint metadata
22:50:34.457 [ajour_core::parse][DEBUG] retail - 33 addons with curse id
22:50:34.911 [ajour_core::parse][DEBUG] retail - 33 addons applied with curse id package metadata
22:50:34.911 [ajour_core::parse][DEBUG] retail - 35 addons successfully parsed
22:50:34.913 [ajour::gui::update][DEBUG] Message::ParsedAddons(retail, 35 addons)
22:50:36.542 [ajour::gui::update][DEBUG] Interaction::Refresh
22:50:36.542 [ajour_core::config][DEBUG] loading config
22:50:36.544 [ajour::gui::update][DEBUG] Message::Parse
22:50:36.544 [ajour::gui::update][DEBUG] config loaded:
Config {
    wow: Wow {
        directory: Some(
            "D:\\Games\\World of Warcraft",
        ),
        flavor: Retail,
    },
    addons: Addons {
        ignored: {
            Retail: [],
        },
        release_channels: {
            Retail: {},
        },
    },
    theme: None,
    column_config: V1 {
        local_version_width: 150,
        remote_version_width: 150,
        status_width: 85,
    },
    window_size: Some(
        (
            645,
            544,
        ),
    ),
    scale: None,
    backup_directory: None,
}
22:50:36.544 [ajour::gui::update][DEBUG] preparing to parse addons in "D:\\Games\\World of Warcraft\\_retail_\\Interface/AddOns"
22:50:36.544 [ajour::gui::update][DEBUG] preparing to parse addons in "D:\\Games\\World of Warcraft\\_classic_\\Interface/AddOns"
22:50:36.544 [ajour_core::parse][DEBUG] retail - parsing addons folder
22:50:36.544 [ajour_core::parse][DEBUG] classic - parsing addons folder
22:50:36.544 [ajour_core::parse][DEBUG] retail - 102 folders in AddOns directory to parse
22:50:36.545 [ajour::gui::update][ERROR] Message::ParsedAddons(classic) - Addon directory not found: "D:\\Games\\World of Warcraft\\_classic_\\Interface/AddOns"
22:50:36.547 [ajour_core::parse][DEBUG] retail - 102 fingerprints: 102 cached, 0 calculated, 0 added, 0 removed
22:50:36.578 [ajour_core::parse][DEBUG] retail - 101 successfully parsed from '.toc'
22:50:36.578 [ajour_core::parse][DEBUG] retail - 2 addons with tukui id
22:50:37.347 [ajour_core::parse][DEBUG] retail - 2 addons applied with tukui id package metadata
22:50:37.347 [ajour_core::parse][DEBUG] retail - 99 unique fingerprints to check against curse api
22:50:37.638 [ajour_core::parse][DEBUG] retail - 38 exact fingerprint matches against curse api
22:50:37.639 [ajour_core::parse][DEBUG] retail - 38 addons applied with fingerprint metadata
22:50:37.639 [ajour_core::parse][DEBUG] retail - 38 addons with curse id
22:50:38.383 [ajour_core::parse][DEBUG] retail - 38 addons applied with curse id package metadata
22:50:38.383 [ajour_core::parse][DEBUG] retail - 40 addons successfully parsed
22:50:38.385 [ajour::gui::update][DEBUG] Message::ParsedAddons(retail, 40 addons)
22:50:39.803 [ajour::gui::update][DEBUG] Interaction::Refresh
22:50:39.804 [ajour_core::config][DEBUG] loading config
22:50:39.806 [ajour::gui::update][DEBUG] Message::Parse
22:50:39.806 [ajour::gui::update][DEBUG] config loaded:
Config {
    wow: Wow {
        directory: Some(
            "D:\\Games\\World of Warcraft",
        ),
        flavor: Retail,
    },
    addons: Addons {
        ignored: {
            Retail: [],
        },
        release_channels: {
            Retail: {},
        },
    },
    theme: None,
    column_config: V1 {
        local_version_width: 150,
        remote_version_width: 150,
        status_width: 85,
    },
    window_size: Some(
        (
            645,
            544,
        ),
    ),
    scale: None,
    backup_directory: None,
}
22:50:39.806 [ajour::gui::update][DEBUG] preparing to parse addons in "D:\\Games\\World of Warcraft\\_retail_\\Interface/AddOns"
22:50:39.807 [ajour::gui::update][DEBUG] preparing to parse addons in "D:\\Games\\World of Warcraft\\_classic_\\Interface/AddOns"
22:50:39.807 [ajour_core::parse][DEBUG] retail - parsing addons folder
22:50:39.807 [ajour_core::parse][DEBUG] classic - parsing addons folder
22:50:39.807 [ajour_core::parse][DEBUG] retail - 102 folders in AddOns directory to parse
22:50:39.808 [ajour::gui::update][ERROR] Message::ParsedAddons(classic) - Addon directory not found: "D:\\Games\\World of Warcraft\\_classic_\\Interface/AddOns"
22:50:39.811 [ajour_core::parse][DEBUG] retail - 102 fingerprints: 102 cached, 0 calculated, 0 added, 0 removed
22:50:39.844 [ajour_core::parse][DEBUG] retail - 101 successfully parsed from '.toc'
22:50:39.844 [ajour_core::parse][DEBUG] retail - 2 addons with tukui id
22:50:40.627 [ajour_core::parse][DEBUG] retail - 2 addons applied with tukui id package metadata
22:50:40.627 [ajour_core::parse][DEBUG] retail - 99 unique fingerprints to check against curse api
22:50:41.002 [ajour_core::parse][DEBUG] retail - 35 exact fingerprint matches against curse api
22:50:41.002 [ajour_core::parse][DEBUG] retail - 35 addons applied with fingerprint metadata
22:50:41.002 [ajour_core::parse][DEBUG] retail - 35 addons with curse id
22:50:41.384 [ajour_core::parse][DEBUG] retail - 35 addons applied with curse id package metadata
22:50:41.385 [ajour_core::parse][DEBUG] retail - 37 addons successfully parsed
22:50:41.387 [ajour::gui::update][DEBUG] Message::ParsedAddons(retail, 37 addons)

**Comment by casperstorm** [comment]
This is unfortunately a issue because the CurseForge api returns garbage right now (see #239).

Short shorty:
You ask the API two questions they answer two other questions. Then you ask the API the same two questions, this time it answer one of them. Rinse repeat. This is why you can press refresh and see different results from time to time. It is likely due to their caching server failing.

TL;DR: We have to wait until CurseForge fixes their api.
